src/core/connectors.js

```diff
@@ -318,9 +318,9 @@ const connectors = [{
 	js: 'connectors/tidal.js',
 	id: 'tidal',
 }, {
-	label: 'Hype Machine Premieres',
+	label: 'Hype Machine Featured Albums',
 	matches: [
-		'*://hypem.com/premiere/*',
+		'*://hypem.com/album/*',
 	],
 	js: 'connectors/hypem-premieres.js',
```

Comment:
I think it should be updated too.

Reply:
Just the filename or the `id` as well? If `id` is changed, should I write a migration to keep options set for it?

Reply:
No, I meant just the filename. ID should be kept as is for compatibility.